### DISCUSSION TITLE
generating a pdf

### DISCUSSION THREAD

**Discussion by dkebler** [thegeeklab/hugo-geekdoc#395] [Ideas]
This repo will take a hugo site and make a pdf from it.  
https://github.com/jgazeau/website2pdf

It was originally created for doing such to hugo sites.
https://discourse.gohugo.io/t/generate-hugo-website-as-a-pdf/22855/15

but templates must use https://www.sitemaps.org/protocol.html this protocol

Wondering how much work it would be to modify geekdoc templates to comply?  

I've forked and been using my flavor of geekdoc for awhile so I am familiar with the templates.  The geekdoc menu filetree navigation to a pdf toc would be very cool.

**Comment by xoxys** [comment]
Converting Websites to a proper PDF were always a pain to me, and I was never happy with the results. Don't know how much time it would need to get it working with the theme. It's not really clear to me what really need to be done. If only a proper sitemap is required, that's pretty easy to achieve with Hugo.
